src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
I agree with the sentiment, but I don't think we should do all kinds of refactoring in this change.

Do you think I should keep it as is, or do something like

     const int frames_per_second = SAMPLE_RATE / iFrameSize;

above it?

Or something else?